coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -972,6 +973,20 @@ class MultiPartitionPlanner(val partitionLocationProvider: PartitionLocationProv
     PlanResult(Seq(resPlan))
   }
   //scalastyle:on method.length
+
+  def supportRemoteRawExport(lp: LogicalPlan): Boolean = {
+    val matchingColumnNameForDisabledTenants = queryConfig.routingConfig.stitchDisabledTenantColumn
```

Comment:
Maybe something like `tenantIdentifierColumn`? Then below could be e.g. `tenantIdentifiers`.

Reply:
I know the name is awkward but ``tenantIdentifierColumn`` doesn't tell column is used for.